feature-libs/organization/administration/components/shared/detail/explain-disable-messages/explain-disable-messages.component.spec.ts

```diff
@@ -0,0 +1,225 @@
+      const element = fixture.debugElement.queryAll(
+        By.css('section > ul > li')
+      )[1].nativeElement;
+      expect(element.innerText).not.toContain(
+        'orgUnit.messages.disabledEnable'
+      );
+    });
+  });
+
+  describe('displayMessageConfig', () => {
+    const displayMessageConfigTrue = [
+      {
+        key: 'disabledEdit',
+        currentValue: { active: false, orgUnit: { active: true } },
+        displayMessageConfig: { disabledEdit: true },
+        expectedValue: ['orgUnit.messages.disabledEdit'],
+      },
+      {
+        key: 'disabledEnable',
+        currentValue: { orgUnit: { active: false } },
+        displayMessageConfig: { disabledEnable: true },
+        expectedValue: [
+          'orgUnit.messages.disabledEdit',
+          'orgUnit.messages.disabledEnable',
+        ],
+      },
+      {
+        key: 'disabledCreate',
+        currentValue: { active: false },
+        displayMessageConfig: {
+          disabledEdit: false,
+          disabledEnable: false,
+          disabledCreate: true,
+        },
+        expectedValue: ['orgUnit.messages.disabledCreate'],
+      },
+      {
+        key: 'disabledDisable',
+        currentValue: {
+          uid: 'test',
+          name: 'test',
+          parentOrgUnit: 'test',
+          active: true,
+        },
+        displayMessageConfig: {
+          disabledDisable: true,
+        },
+        expectedValue: ['orgUnit.messages.disabledDisable'],
+      },
+    ];
+    const displayMessageConfigFalse = [
+      {
+        key: 'disabledEdit',
+        currentValue: { active: true, orgUnit: { active: true } },
+        displayMessageConfig: { disabledEdit: true },
+        expectedValue: ['orgUnit.messages.disabledEdit'],
+      },
+      {
+        key: 'disabledEnable',
+        currentValue: { orgUnit: { active: true } },
+        displayMessageConfig: { disabledEnable: true },
+        expectedValue: [
+          'orgUnit.messages.disabledEdit',
+          'orgUnit.messages.disabledEnable',
+        ],
+      },
+      {
+        key: 'disabledCreate',
+        currentValue: { active: true },
+        displayMessageConfig: {
+          disabledEdit: false,
+          disabledEnable: false,
+          disabledCreate: true,
+        },
+        expectedValue: ['orgUnit.messages.disabledCreate'],
+      },
+      {
+        key: 'disabledDisable',
+        currentValue: { active: true, parentOrgUnit: 'test' },
+        displayMessageConfig: {
+          disabledDisable: true,
+        },
+        expectedValue: ['orgUnit.messages.disabledDisable'],
+      },
+    ];
+
+    displayMessageConfigTrue.forEach((action) => {
+      it(`should display ${action.key} message when ${action.key} is enabled`, () => {
+        component.displayMessageConfig = action.displayMessageConfig;
+        component.ngOnInit();
+        current$.next(action.currentValue);
+        fixture.detectChanges();
+        const element = fixture.debugElement
```

Comment:
its no longer element, its array of string now
```suggestion
        const values = fixture.debugElement
```